core/templates/dev/head/domain/collection/CollectionRightsBackendApiService.js

```diff
@@ -67,6 +150,22 @@ oppia.factory('CollectionRightsBackendApiService', [
             _setCollectionStatus(
               collectionId, collectionVersion, false, resolve, reject);
           });
+        },
+
+        /**
+         * Returns true if the collection is private given the collectionId.
+         * This method will check the collection rights object exists, if so,
```

Comment:
Nit: ``This method will check whether the collection rights object exists and, if so, will return the result from the cache.``